GraphicEditor/ViewModel/FloorUserControlViewModel.cs

```diff
@@ -7,13 +8,14 @@
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Media;
+using System.Windows.Threading;
 
 namespace GraphicEditor.ViewModel
 {
     public class FloorUserControlViewModel : BindableBase
     {
         public ResourceDictionary ResourceDictionary = new ResourceDictionary();
-        public Dictionary<string, RoomButton> connections = new Dictionary<string, RoomButton>();
+        public Dictionary<string, RoomButton> connections;  //= new Dictionary<string, RoomButton>();
```

Comment:
Idk if this is left in the comment by mistake or if it's necessary, just check it out 😄

Reply:
I could have deleted this, but it was due to trying something out, and when I realized that it worked without this, I forgot to delete this.